src/components/Login/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { FilterProvider } from "../../contexts/FilterContext";
+import HeaderMinimal from "../HeaderMinimal";
+import LoginBody from "../LoginBody";
+
+const Login = () => (
+  <FilterProvider>
+    <HeaderMinimal />
+    <LoginBody />
+  </FilterProvider>
+);
```

Comment:
You don't have to wrap every component with filter provider, wrap Routes in the App with provider

Reply:
corrected